ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/SeverityCountLabels.tsx

```diff
@@ -21,9 +21,9 @@ function getTooltipContent(severity: string, severityCount?: number, entity?: st
         return `${capitalize(severity)} severity is hidden by the applied filter`;
     }
     if (entity) {
-        return `${pluralize(severityCount, `${severity} severity CVE`)} across this ${entity}`;
+        return `${severityCount} ${severity} severity cve count across this ${entity}`;
```

Comment:
Worth discussing at joint meeting on Wednesday: plurality problem and possible solutions.

For my understanding:
* Is the number in this case for the tooltip more specific than a count rendered on the page?
* Noticing lowercase cve reminded me of recent discussion about pro and con of cve versus vulnerability as generic term. Is that also worth to discuss at joint meeting?

Reply:
As an afterthought, where rewrite of classic code seemed out of scope, an alternative solution worth to keep in mind is cypress `contains` method with regular expression.

Reply:
> Is the number in this case for the tooltip more specific than a count rendered on the page?

This is a good question. I'd like to check this with an actual screenreader to see how it sounds, given that the count itself is present in the text. It is possible that the count is double-mentioned, making this redundant here.

> Noticing lowercase cve reminded me of recent discussion about pro and con of cve versus vulnerability as generic term. Is that also worth to discuss at joint meeting?

Yes, worth discussing at the joint meeting. There were even comments in the VM sync the other day questioning the use of "CVE" in the toggle button, given that the list below contains both CVEs and RHSAs.